File: S03-conceptos-basicos/CL20-RecyclerViewAdapter_CardView/app/src/main/java/com/udemyandroid/recyclerview/MainActivity.java
package com.udemyandroid.recyclerview;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity implements RestauranteFragment.OnListFragmentInteractionListener {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    @Override
    public void onListFragmentInteraction(Restaurante item) {

    }
}

File: S13-kotlin/CL09-DataBindingData/app/src/main/java/com/androidavanzado/CL09databinding/MainActivity.kt
package com.androidavanzado.CL09databinding

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.databinding.DataBindingUtil
import com.androidavanzado.CL09databinding.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = DataBindingUtil.setContentView(this, R.layout.activity_main)

        var usuario: Usuario = Usuario()
        usuario.nombre = "Miguel"
        usuario.email = "info@miguelcampos.dev"

        binding.user = usuario
    }
}
